BDInfo/BDROM.cs

```diff
@@ -164,7 +164,7 @@ public class BDROM
             if (DirectoryPLAYLIST != null)
             {
                 FileSystemMetadata[] files = GetFiles(DirectoryPLAYLIST.FullName, ".mpls").ToArray();
-                foreach (FileSystemMetadata file in files)
+                foreach (var file in files)
```

Comment:
What are the var settings? I hope it only does this in the cases where the initialization is really obvious such as this. :)